.github/workflows/docker-build-api-executors-tag.yaml

```diff
@@ -890,6 +844,82 @@ jobs:
           DOCKER_BUILDX_CACHE_FROM: "type=gha"
           DOCKER_BUILDX_CACHE_TO: "type=gha,mode=max"
 
+  log_server_sidecar:
```

Comment:
nit: we use kebab-case everywhere, I'd keep it uniform

Reply:
hm, in all workflows I see `snake case`, should I change everything to kebab-case ?